lib/Plack/Middleware/Recorder.pm

```diff
@@ -22,13 +22,26 @@ sub prepare_app {
     $self->start_url('/recorder/start') unless defined $self->start_url;
     $self->stop_url('/recorder/stop')   unless defined $self->stop_url;
 
-    my $output = $self->{'output'};
+    my $output = delete $self->{output};
     croak "output parameter required" unless defined $output;
 
-    $output = $self->{'output'} = IO::File->new($output, 'w') or croak $!
-        unless ref $output;
+    if (ref $output) {
```

Comment:
@brummett I was looking at the commit that introduced this, and I'm not sure why you lazily initialize the output file handle.  Does doing it here interfere with CGI somehow?